### ISSUE TITLE
Start the process of following our contributors on Twitter!

### ISSUE THREAD

**Issue by fharper** [meilisearch/devrel#184]
Need to do some following for past ones.

**Comment by fharper** [comment]
It's a small gesture that will show them that we care.

@CaroFG since you have a list of people you send stickers to, I assume I could just use this list to see who we need to follow?

**Comment by CaroFG** [comment]
Yep, or you can search for the stickers-2021 or will-send-stickers-2021 tags on Orbit :)
